projects/dsp-ui/src/lib/viewer/values/interval-value/interval-input/interval-input.component.ts

```diff
@@ -157,6 +183,16 @@ export class IntervalInputComponent extends _MatInputMixinBase implements Contro
         }
     }
 
+    ngOnInit() {
+        if (this.valueRequiredValidator) {
+            this.startIntervalControl.setValidators([Validators.required, startEndSameTypeValidator(this.endIntervalControl)]);
```

Comment:
Would it make sense to add the `startEndSameTypeValidator` also when the required validator is not set?

Reply:
Definitely, I forgot. Added in 3c0a216